libs/ui/package.json

```
@@ -0,0 +1,79 @@
+{
+  "name": "@votingworks/ui",
+  "version": "1.0.0",
+  "private": true,
+  "description": "Generic UI components for VotingWorks UIs",
+  "license": "GPL-3.0",
+  "author": "Caroline Modic",
```

I probably have one with my name on it, but I intend to change any to this:
```suggestion
  "author": "VotingWorks <eng@voting.works>",
```